### PR TITLE
Bug 1486938 Added nodeselector to metrics docs

### PR DESCRIPTION
**Pull request by bfallonf**
For: https://bugzilla.redhat.com/show_bug.cgi?id=1486938

@thoraxe PTAL

### CODE REVIEW

**Comment by bfallonf** [comment]
Verified in the BZ. @openshift/team-documentation PTAL

File: install_config/cluster_metrics.adoc [hunk 1]
```diff
@@ -357,6 +357,12 @@ millicores) would request 4 CPUs.
 |`openshift_metrics_cassandra_storage_group`
 |The supplemental storage group to use for Cassandra.
 
+|`openshift_metrics_cassandra_nodeselector`
+|Set to the desired, existing
+xref:../admin_guide/scheduling/node_selector.adoc[node selector] to ensure that
```
Comment (ahardin-rh) [inline install_config/cluster_metrics.adoc]: Need an anchor after the .adoc (same for all instances on the page). Everything else looks great 👍

**Comment by bfallonf** [comment]
Thanks @ahardin-rh . Fixed. I'll merge.

**Comment by bfallonf** [comment]
[rev_history]
|xref:../install_config/cluster_metrics.adoc#install-config-cluster-metrics[Enabling Cluster Metrics]
|Added `*_nodeselector` variables to the table in the xref:../install_config/cluster_metrics.adoc#metrics-ansible-variables[Specifying Metrics Ansible Variables] section.
%

**Pull request merged**